ports/psoc6/mtb-libs/mpy-psoc.ld

```diff
@@ -0,0 +1,390 @@
+/* 
```

Comment:
This .ld file is valid for all the mcu and boards? or it should be only be enabled in the AI kit makefile?

Reply:
Right now it is valid for all the boards enabled because the starting locations in all cores is same and allocations are within permissible ranges. We could further optimize of BLE board but only if needed.